style.css

```diff
@@ -0,0 +1,391 @@
+.profileImage h2 {
+	font-size: 3rem;
+	margin: 0;
+	padding-bottom: 20px;
+	
+}
+
+.profileImage img {
+	width: 300px;
+}
+
+section p {
+	font-size: 2rem;
+	padding: 120px 0px 80px 80px;
+}
+
+main {
+	display: flex;
+	padding: 0px 120px 80px 120px;
+	background-color: #fef5ec;
+}
+
+
+.projectsIntro {
+	padding: 20px 140px;
+	background-color: #fef5ec;
+}
+
+.projectsIntro h2 {
+	margin: 0;
+	padding: 40px 0px 0px 0px;
+	font-size: 3rem;
+}
+
+.projectBox {
+	width: 33%;
+	background-color: #c7f4f8;
+	margin-left: 20px;
+	margin-right: 20px;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	padding: 10px 30px 30px 30px;
+}
+
+.projectBox h3 {
+	width: 100%;
+	text-align: center;
+	font-size: 2rem;
+}
+
+.projectBox img {
+	width: 80%;
+}
+
+.projectBox p {
+	font-size: 1.8rem;
+	text-align: center;
+	padding: 20px;
+	margin: 0;
+}
+
+.projectLinks a {
+	margin-right: 20px;
+	text-decoration: none;
+	color: #002d5d;
+	font-size: 1.4rem;
+}
+
+.projectLinks a:hover {
+	color: #2eccd3;
+}
+
+.contact {
+	width: 100%;
+	height: 300px;
+	
+	background-color: #c7f4f8;
+}
+
+.contact h2 {
+	padding: 60px 140px 0px 140px;
+	margin: 0;
+	font-size: 3rem;
+}
+
+.contactLinks {
+	padding: 0px 140px;
+	
+}
+
+.contactLinks a {
+	margin-right: 30px;
+	font-size: 4rem;
+	color: #7f52a0;
+}
+
+.contactLinks a:hover {
+	color: #2eccd3;
+}
+
+.contact p {
+	font-size: 2rem;
+	padding: 20px 140px 30px 140px;
+	margin: 0;
+	
+}
+
+footer {
+	width: 100%;
+	height: 50px;
+	background-color: #fef5ec;
+	
+}
+
+footer p {
+	padding: 20px 140px;
+	margin: 0;
+	color: #002d5d;
+}
+
+
+
+@media only screen and (max-width: 375px) {
+	
+	nav {
+		padding: 20px 10px;
+	}
+	
+	nav, nav a {
+		font-size: 1.2rem;	
+	}	
+	
+	.about { 
+		padding: 40px 10px;
+	}
+	
+	.profileImage {
+		text-align: center;
+	}
+	
+	section p {
+		font-size: 1.8rem;
+		padding: 5px;
+	}
+	
+	main {
+	    display: flex;
+		flex-wrap: wrap;
+		padding: 0px 15px 50px 15px;	
+	}
+	
+	.contact p {
+		padding: 20px 100px 30px 100px;
+		
+	}
+	
+	.contactLinks {
+		padding: 0px 60px;
+	}
+	
+	
+	.contactLinks a {
+		margin: 10px;
+	}
+	
+	footer {
+		height: 90px;
+	}
+	
+	footer p {
+		padding: 40px 140px;
+	}
+	
+}
+
+
+@media only screen and (max-width: 500px) {
+	
+	.about { 
+		padding: 40px 20px;
+	}	
+	
+	main {
+		padding: 0px 15px 50px 15px;	
+}
+
+	.contactLinks {
+		padding: 0px 20px;
+	}
+	
+	.contact p {
+		padding: 20px 60px 40px 60px;
+		
+	}
+}
+
+
+@media only screen and (max-width: 400px) {
+	
+	nav, nav a {
+		font-size: 1.3rem;	
+	}	
+	
+	main {
+		padding: 0px 15px 30px 15px;	
+	}
+	
+	.contactLinks {
+		padding: 0px 40px;
+	}
+	
+	.contact p {
+		padding: 20px 60px 40px 60px;
+		
+	}
+	
+}
+
+
+@media only screen and (max-width: 800px) {
+	header {
+		height: 600px;
+		display: flex;
+		justify-content: center;	
+	
+	}
+}
+
+
+@media only screen and (max-width: 1000px) {
+	
+	nav, nav a {
+		font-size: 1.5rem;	
+	}	
+	
+	header {
+		height: 600px;
+		display: flex;
```

Comment:
display flex and justify content arent needed here as they are already included above